gradle.properties

```diff
@@ -4,6 +4,9 @@ projectGroup=org.eclipse.dataspaceconnector.registrationservice
 
 edcGroup=org.eclipse.dataspaceconnector
 edcVersion=0.0.1-SNAPSHOT
+identityHubGroup=org.eclipse.dataspaceconnector.identityhub
+identityHubVersion=0.0.1-SNAPSHOT
```

Comment:
just FYI: unless the snapshot repo (`https://oss.sonatype.org/content/repositories/snapshots/`) is added to `build.gradle.kts` this will require a local-checkout and `publishToMavenLocal`.

Reply:
indeed and this local checkout is done in action.yml